### PR TITLE
Add com.github.aharotias2.parapara

### PR DESCRIPTION
**Pull request by aharotias2**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

I'm a little excited because this is my first pull request.
Thank you for giving me this wonderful opportunity.

### CODE REVIEW

File: com.github.aharotias2.parapara.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/elementary/granite.git
+        tag: '6.2.0'
+        commit: 4ab145c28bb3db6372fe519e8bd79c645edfcda3
+  - name: parapara
+    buildsystem: meson
+    sources:
+      - type: dir
```
Comment (hfiguiere) [inline com.github.aharotias2.parapara.yml:22]: you need to actually get the sources here.

File: com.github.aharotias2.parapara.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        path: .
```
Comment (ryonakano) [inline com.github.aharotias2.parapara.yml:23]:
```suggestion
      - type: git
        url: https://github.com/aharotias2/parapara.git
        tag: 'v3.2.1'
        commit: ecc80af99ca8d7fc2811a585f85ba07f7e065fc8
```

FYI I added a diff. And every time you create a new release in your repository replace `tag` and `commit` values.

Note that this changes should be only for releasing for Flathub so you shouldn't modify the original yaml file in your repository which is for development.

**Comment by hfiguiere** [comment]
bot, build com.github.aharotias2.parapara

**Comment by aharotias2** [comment]
@ryonakano
Thank you very much!
I have merged your suggestion.

**Comment by aharotias2** [comment]
@hfiguiere
We have fixed the problem.
check it again please.

**Comment by aharotias2** [comment]
@hfiguiere
I didn't install appdata file, so fixed it.
Could you run the test again, please?
Thank you.

**Comment by hfiguiere** [comment]
bot, build com.github.aharotias2.parapara

**Comment by hfiguiere** [comment]
you can also trigger the build yourself. no special permission needed. (I triggered it this time before you asked)

**Comment by aharotias2** [comment]
bot, build com.github.aharotias2.parapara

**Comment by aharotias2** [comment]
@barthalion
Thank you very much!!

**Pull request closed without merging**